test/live.test.ts

```diff
@@ -7,9 +7,9 @@ import { LiveConnectionState, LiveTranscriptionEvents } from "../src/lib/enums";
 describe("connecting to our transcription websocket", () => {
   let deepgram: DeepgramClient;
 
-  beforeEach(() => {
+  before(() => {
     deepgram = createClient(faker.string.alphanumeric(40), {
-      global: { url: "https://api.mock.deepgram.com" },
+      global: { url: "https://api.mock.deepgram" },
```

Comment:
Not sure if the removal of `.com` is intentional.

Reply:
It is intentional! But the commit of it wasn't lol. thanks!

Reply:
fixed